mopidy/internal/network.py

```diff
@@ -179,13 +189,13 @@ class Connection(object):
     def __init__(self, protocol, protocol_kwargs, sock, addr, timeout):
         sock.setblocking(False)
 
-        if (sock.family == socket.AF_UNIX):
+        if is_unix_socket(sock):
```

Comment:
With the changes you just made to `accept_connection` you don't need this condition at all anymore.

Reply:
Do you mean: 
```
-        if is_unix_socket(sock):
-            self.host = sock.getsockname()
-            self.port = None
-        else:
-            self.host, self.port = addr[:2]  # IPv6 has larger addr
+        self.host, self.port = addr[:2]  # IPv6 has larger addr
```
I think the extra verbosity of the current code makes more sense though. I'll wait to commit and push until you confirm you'd prefer it this way.

Reply:
I personally don't think it needs to worry about this extra special case since it fits in the general case

Reply:
Okay cool I'll push those changes